packages/frontend/src/view/pages/user/UserPage.tsx

```diff
@@ -87,7 +87,7 @@ function UserPage(props: UserPageProps) {
             type={props.type}
             starkKey={props.starkKey}
             assets={props.assets}
-            isMine={props.user !== undefined}
+            isMine={isMine && props.type === 'SPOT'}
```

Comment:
I would suggest to use props.type conditional rendering inside the component not here

Reply:
Fixed